code/game/machinery/gulag_teleporter.dm

```diff
@@ -0,0 +1,129 @@
+/*
+The gulag teleporter teleports a prisoner to the gulag outpost.
+It automatically strips the prisoner and equips a prisoner ID, prisoner jumpsuit and oranges sneakers.
+You can set the amount of points in the console
+The console is located at computer/gulag_teleporter.dm
+*/
+
+//Gulag teleporter
+/obj/machinery/gulag_teleporter
+	name = "labor camp teleporter"
+	desc = "A bluespace teleporter used for teleporting prisoners to the labor camp."
+	icon = 'icons/obj/machines/implantchair.dmi'
+	icon_state = "implantchair"
+	state_open = 0
+	density = 1
+	anchored = 1
+	use_power = 1
+	idle_power_usage = 200
+	active_power_usage = 5000
+	var/locked = 0
+	var/list/stored_items = list()
+	var/jumpsuit_type = /obj/item/clothing/under/rank/prisoner
+	var/shoes_type = /obj/item/clothing/shoes/sneakers/orange
+
+/obj/machinery/gulag_teleporter/interact(mob/user)
+	if(locked)
+		user << "[src] is locked."
+		return
+	toggle_open()
+
+/obj/machinery/gulag_teleporter/updateUsrDialog()
```

Comment:
I'll let this slide, the old system is deprecated and all you're doing here is turning it off for your type.